packages/ast/src/clients/helper-funcs/msg-funcs.ts

```diff
@@ -13,16 +13,60 @@ import { GenericParseContext } from "../../encoding";
 export function createMsgHelperCreator(
     context: GenericParseContext,
     service: ProtoService,
```

Comment:
We can change to ProtoServiceMethod here like the query did here

And change the way of the call to this outside